css/index.css

```diff
@@ -1,3 +1,461 @@
+*/
 
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+/*************************************************
+******                                       *****
+******    These Styles are for every page    *****
+******                                       *****
+**************************************************/
+
+
+.test {
+	border: 2px solid red;
+}
+
+.container {
+	max-width: 1000px;
+	margin: 0 auto;
+	padding: 15px 0;
+	padding-top: 130px;
+}
+
+@media (max-width: 800px) {
+    .container {
+        width: 94%;
+        margin: 0 3%;
+    }
+}
+
+@media (max-width: 500px){
+    .container {
+        margin-top: 270px;
+    }
+}
+
+.banner {
+    position: fixed;
+    width: 100%;
+	top: 0;
+	box-shadow: 2px 0 3px black;
+    background: rgba(255,255,255, 0.7);
+}
+
+@media (max-width: 800px) {
+    .banner {
+        position: absolute;
+        width: 100%;
+        box-shadow: none;
+    }
+}
+
+/* Styling our banner */
+.banner-wrap {
+    display: flex;
+	justify-content: space-between;
+    align-items: center;
+    
+    max-width: 1000px;
+    margin: 0 auto;
+}
+
+@media (max-width: 800px) {
+    .banner-wrap {
+        flex-direction: column-reverse;
+        padding-top: 30px;
+        width: 94%;
+        margin: 0 3%;
+    }
+}
+
+.banner .banner-wrap nav {
+	max-width: 1000px;
+    padding: 15px 0;
+    width: 70%;
+    display: flex;
+    justify-content: space-between;
+}
+
+@media (max-width: 800px) {
+    .banner .banner-wrap nav {
+        width: 95%;
+        margin: 22px auto 0;
+        font-size: 14px;
+    }
+}
+
+@Media (max-width: 500px){
```

Comment:
you probably noticed you have some thick black lines in some places from the borders overlapping in your mobile view nav. I would put a border-top on the nav container here and border-bottom on your a tags